Lib/idlelib/config-keys.def

```diff
@@ -24,7 +24,7 @@ history-previous=<Alt-Key-p> <Meta-Key-p> <Alt-Key-P> <Meta-Key-P>
 interrupt-execution=<Control-Key-c> <Control-Key-C>
 view-restart=<Key-F6>
 restart-shell=<Control-Key-F6>
-open-class-browser=<Alt-Key-c> <Meta-Key-c> <Alt-Key-C> <Meta-Key-C>
+open-module-browser=<Alt-Key-c> <Meta-Key-c> <Alt-Key-C> <Meta-Key-C>
```

Comment:
Changing the virtual event name breaks compatibility existing user customizations.  I don't want to do that, certainly not yet.  So revert all in this file.